Modules/Segmentation/ConnectedComponents/test/itkRelabelComponentImageFilterSignednessTest.cxx

```diff
@@ -0,0 +1,52 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkImage.h"
+#include "itkRelabelComponentImageFilter.h"
+#include "itkTestingMacros.h"
+
+
+int
+itkRelabelComponentImageFilterSignednessTest(int, char *[])
+{
```

Comment:
Thanks for investigating this issue, Jon. 👍 It still seems to me that a runtime test is not really necessary, when we _just_ want to check that signed pixel types compile well. Please consider to replace the entire test function with a single explicit template instantiation:

```cpp
// Explicit template instantiation, to test compile-time support of signed pixel types: 
template class itk::RelabelComponentImageFilter<itk::Image<short>, itk::Image<short>>;
```

What do you think?

Reply:
Done in a9fe742.